requirements/requirements-dev.txt

```diff
@@ -1,4 +1,6 @@
+accelerate
 coverage
+diffusers
```

Comment:
should we include these deps into separate requirements? otherwise, it'll keep showing as optional dep into PyPI since we export the [dev] optional package cc @edgarriba

Reply:
Yes